message-index/messages/GHC-64725/encode/after/SomeEncodingLibrary.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
This file is identical to the `before` version, since it represents a module from a library which the user can't (easily) change. Ideally there would be a single `SomeEncodingLibrary.hs` directly at `message-index/messages/GHC-64725/encode/`, but then `ghc` on `Main1.hs` wouldn't see this module.

The same is true of `SomeDisplayLibrary.hs` in the `display` example.